st2actions/st2actions/container/base.py

```diff
@@ -93,10 +95,25 @@ def _do_run(self, runner, runnertype_db, action_db, liveaction_db):
             runner_params, action_params = param_utils.get_finalized_params(
                 runnertype_db.runner_parameters, action_db.parameters, liveaction_db.parameters)
             runner.runner_parameters = runner_params
-            LOG.debug('Performing pre-run for runner: %s', runner)
+
+            LOG.debug('Performing pre-run for runner: %s', runner.runner_id)
             runner.pre_run()
 
-            LOG.debug('Performing run for runner: %s', runner)
+            # Mask secret parameters in the log context
+            runner_parameters_specs = runnertype_db.runner_parameters
+            action_parameters_sepcs = action_db.parameters
+
+            secret_runner_parameters = get_secret_parameters(parameters=runner_parameters_specs)
```

Comment:
Yeah, I agree.

This whole method would need some refactoring love, but I will just refactor the logging stuff for now.